api/notd/store/retriever.py

```diff
@@ -40,6 +39,16 @@ async def generate_token_transfers(self, fieldFilters: Optional[Sequence[FieldFi
             tokenTransfer = token_transfer_from_row(row)
             yield tokenTransfer
 
+    async def normalize_bad_token_transfers(self,fieldFilters: Optional[Sequence[FieldFilter]] = None, length: Optional[int] = 42) -> AsyncGenerator[TokenTransfer, None]:
```

Comment:
this shouldnt go here it should be in the one-off script. no new transfers will be bad so this code only applies to historicl things so it wont needed be in production